nss-day-cohort-45/nutyshell-running-with-scissors-1
# Ar new event form

# Description
Added form for creating new events for the current user.
Fixes # (issue)
## Type of change
# Testing Instructions
- serve database and app
- log in
- create a new event and ensure that it populates in the event list
# Checklist:
- [] I have commented my code, particularly in hard-to-understand areas
